manifest.json

```diff
@@ -1,15 +1,22 @@
 {
+  "$schema": "https://json.schemastore.org/chrome-manifest",
   "manifest_version": 3,
   "name": "__MSG_extensionName__",
   "description": "__MSG_extensionDescription__",
   "version": "1.26.0",
   "version_name": "1.26.0-prerelease",
   "default_locale": "en",
+  "minimum_chrome_version": "93",
   "background": {
-    "service_worker": "background/sw.js"
+    "service_worker": "background/sw.js",
+    "type": "module"
   },
   "action": {
-    "default_icon": "images/icon.png",
+    "default_icon": {
+      "16": "images/icon-16.png",
+      "32": "images/icon-32.png",
+      "1024": "images/icon.png"
+    },
```

Comment:
Why? @TheColaber

Reply:
It was recommended by the schema, but revert if wanted.